src/App.tsx

```diff
@@ -1,85 +1,73 @@
-import React from 'react';
+/* eslint-disable */
+import { useState } from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
 
-export const goods = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
+export const goods = ['Dumplings', 'Carrot', 'Eggs',
+  'Ice cream', 'Apple', 'Bread', 'Fish', 'Honey', 'Jam', 'Garlic'];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [selectedGoods, setSelectedGoods] = useState<string[]>([]);
```

Comment:
as per checklist point number 2 
`Save a selectedGood in the state (Jam is the default value).`